lib/domain-shared/shared_use_cases/account_use_case.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:seeds/datasource/remote/firebase/firebase_message_token_repository.dart';
+
+abstract class AccountUseCase {
+  Future<void> updateFirebaseToken({required String oldAccount, String newAccount = ""}) async {
```

Comment:
this newAccount should be nullable instead of empty, and check for nullability

Reply:
But what if it recive an empty string? Is possible an empty string arrives here?

Reply:
i see.Then dont pre set to ""
String newAccount = ""

remove the default

Reply:
Huh? The standard is that newAccount = "" when it's empty. That's what has been returned from the settings too, the settings never return null. 

I don't understand this.

In our previous agreement settingStorage either has an account name or it has ""

Changing this for this PR is dangerous and introduces problems. 

Given that it is always set in settingStorage, I find it weird to pretend like we're setting it to null here.

settingStorage returns "" when it's set to null.

Reply:
Trying to fix a bug here, not change the entire code base - this topic unrelated to this PR.